tests/callbacks/test_plotting_callback.py

```diff
@@ -42,3 +44,56 @@ def test_show_plot_only_shows_if_backend_is_genie():
         mock_get_backend.return_value = "simulated_genie_python_backend"
         lp._show_plot()
         mock_plt_show.assert_called_once()
+
+
+def test_errorbars_created_if_yerr_is_given():
+    _, ax = plt.subplots()
+    ax.errorbar = MagicMock()
+
+    lp = LivePlot(y="y", x="x", yerr="yerr", ax=ax)
+
+    x = 1
+    y = 2
+    yerr = 3
+
+    # Fake just enough of an event document.
+    lp.start(
+        {
+            "time": 0,
+            "plan_name": "scan",
+            "uid": 0,  # type: ignore
+            "scan_id": 0,
+        }
+    )
+
+    # Fake just enough of an event document.
+    lp.event(
+        {
+            "data": {
+                "y": y,
+                "x": x,
+                "yerr": yerr,
+            }
+        }  # type: ignore
+    )
+
+    lp.ax.errorbar.assert_called_with(y=[y], x=[x], yerr=[yerr], fmt="none")  # type: ignore
+
+
+def test_errorbars_not_created_if_no_yerr():
+    _, ax = plt.subplots()
+    ax.errorbar = MagicMock()
+
+    lp = LivePlot(y="y", x="x", ax=ax)
+
+    lp.start(
+        {
+            "time": 0,
+            "plan_name": "scan",
+            "uid": 0,  # type: ignore
+            "scan_id": 0,
+        }
+    )
+
+    lp.update_plot()
+    assert not lp.ax.errorbar.called  # type: ignore
```

Comment:
Could: I think if you make this just `ax.errorbar` you probably don't need the type ignore?

Reply:
Cannot access attribute "ax" for class "LivePlot". Might have to leave this one